src/actions/roles.js

```diff
@@ -33,32 +30,16 @@ export function getRoleByName(roleName: string) {
             roleName,
         ],
     });
-}
-
-export function getRole(roleId: string) {
-    return bindClientFunc({
-        clientFunc: Client4.getRole,
-        onRequest: RoleTypes.ROLE_BY_ID_REQUEST,
-        onSuccess: [RoleTypes.RECEIVED_ROLE, RoleTypes.ROLE_BY_ID_SUCCESS],
-        onFailure: RoleTypes.ROLE_BY_ID_FAILURE,
-        params: [
-            roleId,
-        ],
-    });
-}
 
-export function editRole(role: Role) {
```

Comment:
Can you add the action here back? As i was saying it is better to have the actions in the file. Just removing the `onRequest`, `onSuccess` and `onFailure` is good enough